execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -309,8 +316,18 @@ impl LmdbWasmTestBuilder {
         path
     }
 
+    /// Returns the file size on disk of the backing lmdb file behind LmdbGlobalState.
+    pub fn lmdb_on_disk_size(&self) -> Option<u64> {
```

Comment:
```suggestion
    pub fn db_on_disk_size(&self) -> Option<u64> {
```

We don't want to "leak" the implementation details which is the usage of LMDB. It's even more true when you replace it with RocksDB.